1-Docker/scripts/set-hosts-headnode.sh

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I really like that you've used awk here, Konstantin Radostev will be very proud of you =)
But jokes aside, what if, let's say, you would need to launch more than one worker in a cluster? Or if I decide to launch worker at first, and then headnode?
Is there a better way to communicate between containers?
Hint: you can already ping from one container to another, even without executing those set-hosts scripts, and without changing /etc/hosts. How?

Reply:
Yes, you're right! There is much easier way to find out IP address of others nodes!

From docker documentation:
> By default Compose sets up a single network for your app. Each container for a service joins the default network and is both reachable by other containers on that network, and discoverable by them at a hostname identical to the container name.

So I can get IP address of other node by using **dig** (from bind-utils), for instance

Here I tried to resolve **headnode** name from **worker**, it works:
![Screenshot_5](https://user-images.githubusercontent.com/40645030/113356588-25be7080-934b-11eb-8e5f-e948aa24461b.png)

So there is no need to add smth to /etc/hosts